### DISCUSSION TITLE
Using norm comand how to I run mutli key shortcuts

### DISCUSSION THREAD

**Discussion by pypp** [vscode-neovim/vscode-neovim#2128] [Q&A] [answered]
For example if I want to increase a number using global command I would run somting like this:

```bash
:g/^\d\+$/normal! ^A
```

using neovim when pressing <kbd>Ctrl</kbd>+<kbd>a</kbd> I would get `^A`, but when using VScode I can't seem to find a way to do that.

**Comment by xiyaowong** [comment, marked as answer]
You need to create a keybinding to send `Ctrl+a` to nvim yourself.

```json
{
 "key": "ctrl+a",
 "args": "<C-a>",
 "command": "vscode-neovim.send"
}
```

**Reply by pypp** [reply]
so do I need to add this to every keybinding there is ?

**Reply by xiyaowong** [reply]
Depending on which keys you need to use.

**Reply by pypp** [reply]
Ok, thanks
